src/Components/Toast/Toast.jsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { toast, ToastContainer } from "react-toastify";
+
+export const ToastLogIn = ({condition}) => {
```

Comment:
u don't need to create  unique toast for every page. Just create few toast(like success, error, e.t.c) and reuse them